Source/ARTPushActivationState.m

```diff
@@ -226,6 +226,8 @@ - (ARTPushActivationState *)transition:(ARTPushActivationEvent *)event {
 
         if ([_fromEvent isKindOfClass:[ARTPushActivationEventCalledActivate class]]) {
             [self.machine callActivatedCallback:nil];
+        } else {
```

Comment:
Is there a test suite that needs to be updated for this behaviour? `PushActivationStateMachineTests` maybe? (I haven't looked in detail at what that file is doing.)

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/f9338a5bcdd5d3bd2681db134496a8b4f07f88e0